specification/monitor/resource-manager/Microsoft.Insights/stable/2022-06-15/actionGroups_API.json

```diff
@@ -0,0 +1,1403 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Action Groups API",
+    "x-ms-code-generation-settings": {
+      "name": "MonitorManagementClient"
+    },
+    "version": "2022-06-15"
+  },
```

Comment:
It is unclear from the PR what the specific changes in this API version are. Can you send the first iteration with the exact cope of the 2022-06-01 API version and add all changes for 2022-06-15 in subsequent iterations of the PR?